src/baseline.es

```diff
@@ -17,9 +17,9 @@ module.exports = Class.extend( {
 		} );
 	},
 
-	merge : function( data, options ) {
-		var options = _.extend( {}, { empty : false }, options );
-		var _this = this;
+	merge( data, defaultOptions ) {
```

Comment:
i would leave this called `options`, since `defaultOptions` to me implies that they are the default but these are the ones that are being passed in

Reply:
Yes! Thanks, its clear now